src/app/visual/fonts/fonts.component.html

```diff
@@ -0,0 +1,13 @@
+<p>
+  Fonts: {{ status }}
+</p>
+
+<ul>
```

Comment:
Thanks for creating these. In addition, would we want to create a visual tests for this, or would all problems with fonts be captured in unit?

Reply:
Good suggestion, I think a visual test would be relevant. I'll add one.